README.md

```diff
@@ -70,7 +70,7 @@ This generates an interactive bash shell for you to run the program in.
 Check out the bot on lichess [here](https://lichess.org/@/Sporkfish)! To run the bot, create a file in the root directory named `api_token.txt`. Add your Lichess bot API token. Then run:
 
 ```
-python3 -O main.py
+python3 main.py
```

Comment:
Sorry, could we keep this pls? This removes runtime asserts, end users don't need to know about them (given we've tested well).